Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
+
+### New features
+
+* Updated the user interface including the following:
+
+  * When joining a game, a "Connecting..." UI loads. When disconnecting from a game, you are returned to the MainMenuScene with a "Connection to Host lost" message. If the game fails to connect, a general message "Connection to Host failed" loads. 
+  * Added an option to leave the lobby after joining it. You no longer have to restart the game to leave the lobby.
+  * Added option to cancel the game after clicking **Ready**. 
+  * Added a gear icon for accessing and modifying app settings. 
+  * The UI now tracks and displays player has arrived, number of connected players in the lobby, and player status. 
+
+* Added F/X and animation assets for the game including:
+
+  * Audio files for boss sound effects
+  * Ramp-up animation for hero movement and actions
+  * Visual effects and particles for Tank charge skill 
+  * Art assets to wave spawner, including animations for ReceiveDamage, Broken (died), and Revive 
+  * Game music 
+
+* Added audiomixer to Boss Room for separate audio channels and a master mixer. 
+
+* Updated and added various hero abilities:
+
+  * Added a cooldown to Archer's PowerShot. 
+  * Used generic actions to implement Rogue's Dagger skill. 
+  * Used generic actions to implement Rogue's Sneak skill using local stealth, applying a graphical effect to the stealthy character while still sending all network events normally. 
+  * Used generic actions to implement Rogue's Dash skill. 
+  * Properly display Heal abilities when targeting a fallen ally character. 
+  * Character attack actions properly support Hold to charge options. 
+
+* To show how UI elements and game objects can be networked, added networked functionality using `INetworkSerializable` for UI elements on the Character Selection screen including networked mouse cursors. 
+* Added a Photon filter for host room names not to allow profanity, racial slurs, and additional socially acceptable words. 
+* Boss Room now uses the [UnityToonShader](https://github.com/IronWarrior/UnityToonShader) for rendering the 3D surfaces to emulate 2D, flat surfaces.
+* Added disconnection error message to load when a player or host disconnects due to limited or no network connectivity. Client logic was also updated to detect Host disconnection scenarios, such as losting connectivity.
+* Balanced hero and enemy stats, spawners, trigger areas, and enemy detetction areas. 
+* Added healthbars to display when damaged and properly track imp health locally and across clients. 
+
+### Changes
+
+* Updated the Photon Setup Guide, indicating you need only app ID when playing with friends. For users connecting across regions, you may need to hard code a region in your app settings by using the room code and region instead of just the room code sharing in game. 
+* Removed Singleton usage, allowing multiple instances of MLAPI networking stack to start up in the same process. 
```

Comment:
Is this referring to MLAPI's `NetworkManager.Singleton`? We haven't yet updated to the newest version that strips the singleton, so we still have `NetworkManager.Singleton` references in BossBoom.

Reply:
Gotcha! thanks!